orchestration/src/main/java/com/sap/ai/sdk/orchestration/client/OrchestrationCompletionApi.java

```diff
@@ -0,0 +1,97 @@
+package com.sap.ai.sdk.orchestration.client;
+
+import com.google.common.annotations.Beta;
+import com.sap.ai.sdk.orchestration.client.model.CompletionPostRequest;
+import com.sap.ai.sdk.orchestration.client.model.CompletionPostResponse;
+import com.sap.cloud.sdk.cloudplatform.connectivity.Destination;
+import com.sap.cloud.sdk.services.openapi.apiclient.ApiClient;
+import com.sap.cloud.sdk.services.openapi.core.AbstractOpenApiService;
+import com.sap.cloud.sdk.services.openapi.core.OpenApiRequestException;
+import java.util.List;
+import javax.annotation.Nonnull;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.util.UriComponentsBuilder;
+
+/**
+ * Internal Orchestration Service API in version 0.0.1.
+ *
+ * <p>SAP AI Core - Orchestration Service API
+ */
+public class OrchestrationCompletionApi extends AbstractOpenApiService {
```

Comment:
since the code is no longer automatically generated, please remove this + the spring dependency. We should add the ability to skip API generation with the Cloud SDK 5.15.0 release as well.

Reply:
I'm removing it